BrainPortal/app/controllers/tasks_controller.rb

```diff
@@ -24,7 +24,7 @@
 class TasksController < ApplicationController
 
   Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
-
+  require 'pry'
```

Comment:
Once again I ask you, can you PLEASE REVIEW YOUR OWN CODE before you commit?

The git utilities in your work computer will show you all changes before you commit locally (use `git diff` and `git diff --staged`), and on top of that, when you make a pull request, the GitHub interface will show you all  lines changed in all files before you actually make the PR. Yes you keep pushing code that changes white space or you leave debug code in your commits. That's unacceptable.

Reply:
yes, sorry forgot to read it. Debug statement is removed now